### ISSUE TITLE
(1.7.2) Unexpected rlp error: RlpIsTooShort (Taking snapshot starting at block 1790000)

### ISSUE THREAD

**Issue by drandreaskrueger** [paritytech/parity#6618]
### problem:

    Taking snapshot at #1790000
    panicked at 'Unexpected rlp error: RlpIsTooShort', src/libcore/result.rs:860

### details:
```
2017-10-02 10:00:02  Imported #1790099 cda7…44d4 (8 txs, 5.47 Mgas, 2.52 ms, 19.75 KiB)
2017-10-02 10:00:19  Imported #1790100 5488…e604 (10 txs, 1.41 Mgas, 2.90 ms, 4.52 KiB)
2017-10-02 10:00:19  Taking snapshot at #1790000
2017-10-02 10:00:19  Taking snapshot starting at block 1790000
thread '<unnamed>' panicked at 'Unexpected rlp error: RlpIsTooShort', src/libcore/result.rs:860
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic
   6: std::panicking::begin_panic_fmt
   7: rust_begin_unwind
   8: core::panicking::panic_fmt
   9: core::result::unwrap_failed
  10: <ethcore::blockchain::blockchain::BlockChain as ethcore::blockchain::blockchain::BlockProvider>::block_receipts
  11: <ethcore::snapshot::consensus::work::PowSnapshot as ethcore::snapshot::consensus::SnapshotComponents>::chunk_all
  12: ethcore::snapshot::chunk_secondary
  13: <F as crossbeam::FnBox>::call_box
  14: std::panicking::try::do_call
  15: <unknown>
```

### versions

```
cat /etc/issue; uname -a; parity --version 
Arch Linux
Linux ... 4.12.4-1-ARCH #1 SMP PREEMPT Fri Jul 28 18:54:18 UTC 2017 x86_64 GNU/Linux
Parity  version Parity/v1.7.2-unstable-9f47909ed-20170918/x86_64-linux-gnu/rustc1.19.0
```

compiled from newest release version `git checkout tags/v1.7.2`

**Comment by arkpar** [comment]
Is that a public blockchain?

**Comment by drandreaskrueger** [comment]
oh, yes, sorry:

    parity --chain ropsten

**Comment by drandreaskrueger** [comment]
I have restarted it, the problem did not re-appear:

```
2017-10-02 11:05:32  Imported #1790386 5470…a401 (2 txs, 1.54 Mgas, 1.01 ms, 8.05 KiB)
2017-10-02 11:05:44  Imported #1790387 7d63…0c34 (0 txs, 0.00 Mgas, 0.25 ms, 0.53 KiB)
```

**Comment by drandreaskrueger** [comment]
the folder

            ~/.local/share/io.parity.ethereum/chains/test/db/ae90623718e47d66/snapshot/

is empty

**Comment by arkpar** [comment]
This is already fixed. The fix is scheduled for 1.7.3
